bosh-director/spec/unit/api/controllers/deployments_controller_spec.rb

```diff
@@ -175,6 +175,17 @@ def manifest_with_errand(deployment_name='errand')
             end
           end
 
+          context 'with the "fix" param' do
+            it 'passes the parameter' do
+              allow_any_instance_of(DeploymentManager)
```

Comment:
Why is this not an `expect` rather than `allow`? Looks like we are actually confirming that the `fix` param is passed into the `DeploymentManager` object.